backstop/tests/header.html

```diff
@@ -609,10 +609,9 @@ <h2>
           class="iui-toggle-switch"
           style="display: inline-flex; vertical-align: middle; margin-left: 1em;"
         >
```

Comment:
Why header test images have changed?

Reply:
The header test images changed because the toggle switch is used to enable / disable slim mode.  This PR removes the padding on top / bottom of the toggle to close #315, which caused the bottom header demo to shift upwards a little bit.